Examples/Google/Language/COMPILE.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
```

Comment:
One of the things we do in `google-api-nodejs-client` that makes reading PRs *much* easier is splitting PRs into two commits - one with the change to the generator code, and another that just contains the generated changes.  We've even toyed with splitting the generator and the generated packages into their own repos to cut down on the noise.

Reply:
I've been wanting to ask you about this: I'd love to split this repo to separate the generator from the generated clients. Let's discuss in person and followup here with our conclusions.